src/components/diagrams/singleLineDiagram/utils.js

```diff
@@ -0,0 +1,168 @@
+/*
```

Comment:
??

Reply:
This file should not exist. I'll check if everything is correct in the new diagram-*.js files